build/instrumentation.build.js

```diff
@@ -101,7 +101,6 @@ export function createConfig({
       ),
       typescript(
         mergeConfig('typescript', {
-          objectHashIgnoreUnknownHack: true,
```

Comment:
Why did you remove the parameter? Just to give some context about the change.

Reply:
Rollup was alerting us to change this flag. If you add it back and run the build, then you will see the warning to remove that warning.